README.md

```diff
@@ -48,10 +51,10 @@ jupyter nbextension enable toc2/main
 
 ## Use
 
-The notebook itself contains detailed usage instructions. 
+The notebooks themselves contains detailed usage instructions. 
 
-Ideally, you will copy this original notebook and use it for each plate you
-create with the shotgun pipeline.
+Ideally, when using a notebook, you will copy the original and 
+use it for each plate you create with the shotgun pipeline.
```

Comment:
Let's change 'each plate you create with the shotgun pipeline' to 'each 96-well plate that is extracted for the abs-quant pipeline.'